multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/BuildApplicationDeployModelStep.java

```diff
@@ -77,20 +77,14 @@ private Staging modifyHealthCheckType(Staging staging) {
         }
         if (HealthCheckType.NONE.getValue()
                                 .equalsIgnoreCase(healthCheckType)) {
-            getStepLogger().info(Messages.USING_DEPRECATED_HEALTH_CHECK_TYPE_USE_ONE_OF_THE_FOLLOWING, healthCheckType,
-                                 getHealthCheckValues());
+            getStepLogger().info(Messages.USING_DEPRECATED_HEALTH_CHECK_TYPE_NONE_SETTING_PROCESS, healthCheckType,
```

Comment:
A bit misleading name, USING_DEPRECATED_HEALTH_CHECK_TYPE_0_SETTING_TO_1 looks better in my opinion.